python/ct/crypto/tools/verify_util.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+"""verify_util.py: CT signature verification utility.
+
+Usage:
+
+  verify_util.py <command> [flags] <cert_file>
```

Comment:
Since this is a generic verify_util, perhaps it won't always have a cert_file? E.g., for verifying STHs.